src/index.html

```diff
@@ -11,8 +14,62 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
Stylesheets with a .scss extension shouldn't be linked directly in HTML. Instead, they should be compiled to CSS and the CSS file should be linked. Make sure you compile your SCSS to CSS and update the href attribute to point to the correct .css file.